stats/views.py

```diff
@@ -114,3 +115,12 @@ def get_data_for_response():
         return {
             'company_count': Fop.objects.count()
         }
+
+class UsersInProjectsView(views.APIView):
+    def get(self, request):
+        user = request.user
+        user_projects = UserProject.objects.filter(user_id=user.id).values_list('project')
```

Comment:
add `flat=True` for better practice
```suggestion
        user_projects = UserProject.objects.filter(
            user_id=user.id
        ).values_list('project', flat=True)
```